webpack.config.ts

```diff
@@ -91,6 +91,10 @@ function ui(_env: unknown, argv: Argv): webpack.Configuration {
           extensions: [".svelte", ".ts", ".js"],
         }),
       ],
+      alias: {
+        svelte: path.resolve("node_modules", "svelte"),
+      },
```

Comment:
Could you explain why you added this? I don’t think it’s necessary.

Reply:
It's required because otherwise Webpack attempts to use the pre-bundled svelte components from `svelte-json-tree`, adding this key means that the svelte source is compiled by our svelte compiler.

Reply:
Isn’t that what the `mainFields` key is for? Could you add comment with a reference so this isn’t removed accidentally?